src/Engine/Party.h

```diff
@@ -360,16 +358,13 @@ struct Party {
     int uNumPrisonTerms;
     unsigned int uNumBountiesCollected;
     int field_74C_set0_unused;
-    std::array<int16_t, 5> monster_id_for_hunting;
-    std::array<int16_t, 5> monster_for_hunting_killed;
+    IndexedArray<int16_t, HOUSE_TOWNHALL_FIRST, HOUSE_TOWNHALL_LAST> monster_id_for_hunting;
```

Comment:
You would think that this would've been 10 too...

Reply:
Yes, but no :)